pubspec.yaml

```diff
@@ -66,7 +68,6 @@ flutter:
   #  - images/a_dot_ham.jpeg
   assets:
     - assets/temp_logo.png
-    - assets/images/
```

Comment:
@bstrumillo I deleted this just an FYI. If you have changes locally commit them so I can track them